packages/phrases/src/locales/en.ts

```diff
@@ -46,6 +46,12 @@ const errors = {
   user: {
     username_exists: 'The username already exists.',
     email_exists: 'The email already exists.',
+    phone_exists: 'The phone already exists.',
+    username_exists_register: 'The username can not be regitsered because it is occupied.',
+    email_exists_register:
+      'The email address can not be regitsered because it is bound to another accournt.',
```

Comment:
`The email address has been registered.`? also for `phone` and `username`

Reply:
fixed.